crates/binstalk/src/fetchers/gh_crate_meta/hosting.rs

```diff
@@ -11,6 +11,18 @@ pub enum RepositoryHost {
     Unknown,
 }
 
+pub const FULL_FILENAMES: &[&str] = &[
+    "{ name }-{ target }-v{ version }.{ archive-format }",
+    "{ name }-{ target }-{ version }.{ archive-format }",
+    "{ name }-{ version }-{ target }.{ archive-format }",
+    "{ name }-v{ version }-{ target }.{ archive-format }",
+];
```

Comment:
point of doing this change set in the original was to reuse the definition for the dirs, to avoid the two getting out of sync. if we're just going to have two arrays (which is fine, just another decision):
- no need to change this, can revert
- let's put a comment on each of these to remember to add to the other when editing

Reply:
I think it's cleaner to have this change in place, so I'd like to keep it.

Reply:
I'd still want the comments as reminder so they don't get out of sync

Reply:
Sure, I will add them.

Reply:
Done